### PR TITLE
Feature/65 create an email class

### PR DESCRIPTION
**Pull request by hitohata**
## Overviews of implementation

added an email address class

## Review points

* I didn't change the UserDto, because I don't think the email address will not be needed for it.

### CODE REVIEW

File: backend/main/src/User/Controllers/CreateUserController.ts [hunk 1]
```diff
@@ -5,6 +5,7 @@ import { CreateUserUseCase } from "src/User/UseCases/CreateUserUseCase/CreateUse
 interface ICreateUserInput {
 	id: string;
 	name: string;
```
Comment (taniguchiiqqq) [inline backend/main/src/User/Controllers/CreateUserController.ts:7]: I'm sorry it's not related to this PR topic directly, but I think this name prop can be optional

**Review by taniguchiiqqq — APPROVED** [review]
LGTM.
I agree that the Dto don't need email field as long as email isn't used for searching user in a group or displaying in user profile page.

File: backend/main/src/User/Controllers/CreateUserController.ts (hunk 1 above)
Comment (hitohata) [reply backend/main/src/User/Controllers/CreateUserController.ts:7]: It should be not needed. So, I deleted it in the "update user ID" PR. lol 🤣 🤣 🤣 🤣

Comment (taniguchiiqqq) [reply backend/main/src/User/Controllers/CreateUserController.ts:7]: Lol. my bad 🤣

**Pull request merged**